### ISSUE TITLE
updates and submitting package to OMF org

### ISSUE THREAD

**Issue by jitakirin** [wa/pkg-keychain#6]
Hi, I [updated the package](https://github.com/jitakirin/pkg-keychain) to work with newer versions of [Oh-My-Fish](https://github.com/oh-my-fish/oh-my-fish) and [fisherman](https://fisherman.github.io/). I also wanted to submit the package to OMF index and request it to be moved into the OMF organisation.

Would you still be interested in ownership and maintenance of the package going forward?

cc: @therealpxc

**Comment by therealpxc** [comment]
It would be nice to have a working plugin in OMF. 👍 

@jitakirin, are you against [using the `env` command to get $SHELL set to "fish" for the purposes of launching `keychain`](https://github.com/therealpxc/dotfiles/blob/master/config/fish/conf.d/keychain.conf.fish) in the plugin instead of advising users to set it globally?

**Comment by jitakirin** [comment]
@therealpxc nope, the thought crossed my mind but then I thought I'll need to set it for docker-machine anyway, but I agree, it would probably be better if things "just worked" :smile:

**Comment by therealpxc** [comment]
Since this appears unmaintained and we haven't heard a response for over a week, do you think we should take the question directly to the oh-my-fish org?
